tests/src/main/java/ee/jakarta/tck/mvc/tests/security/CsrfConstants.java

```diff
@@ -15,9 +15,9 @@
  */
 package ee.jakarta.tck.mvc.tests.security;
 
-public class CsrfConstants {
+import jakarta.mvc.security.Csrf;
 
-    // TODO: Replace with Csrf.DEFAULT_CSRF_HEADER_NAME after updating spec version
-    public static final String CSRF_TOKEN_HEADER_NAME = "X-CSRF-TOKEN";
+public class CsrfConstants {
```

Comment:
Maybe we could completely get rid of this constants class now? Thoughts?

Reply:
Sure why not. I'll remove it